modules/graphics/window-manager.nix

```diff
@@ -0,0 +1,86 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}: {
```

Comment:
You could put `{ config = lib.mkIf (config.ghaf.graphics.weston.enable || config.ghaf.graphics.labwc) { ...` here. Then you can avoid doing that lib.mkMerge in other files.

Reply:
Or simply create own option like `ghaf.graphics.window-manager.enable` and toggle it to true from weston.nix and labwc.nix modules

Reply:
Maybe better name for the module could be like `window-manager-common`

Reply:
Thanks for the hint, addressed